lessons/11-basic-data-structs/notes/hashes.md

```diff
@@ -0,0 +1,95 @@
+# Hashes
+
+A `Hash` is another very popular collection type in Ruby, similar to that Array.
+It is similar to an Array, except that indexing is done via unordered keys of any object type, not an ordered integer index.
+
+```ruby
+{first_num: 1, greeting: "Hello", pi: 3.14}
+```
```

Comment:
Maybe: For example, in this hash declaration we have three key-value pairings. You could compare this to an array like array = [1, "Hello", 3.14]. In the array example, we could access the value 1 by using its index 0, so 
array[0] # ==> 1 
A hash is unordered so we can't access the values by indexing, so instead we have to use keys to access values. In this case we could access the value 1 by using its key first_num, so
asjdfljasd[:first_num] # ==> 1